force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -18,11 +18,39 @@
  */
 trigger ContactTrigger on Contact(before insert) {
 	// When a contact is inserted
-	// if DummyJSON_Id__c is null, generate a random number between 0 and 100 and set this as the contact's DummyJSON_Id__c value
+	// If the contact does not have a value for the DummyJSON_Id__c field, set the value of the field to a random number between 0 and 100 // added to handler class
+	if(Trigger.isBefore && Trigger.isInsert ){
+		for (Contact contact : Trigger.new) {
+			if (contact.DummyJSON_Id__c == '' || contact.DummyJSON_Id__c == null) {
+			contactTriggerHandler.setDummyJSONId(Trigger.new);
+			}
+		}
```

Comment:
Try to remove the logic from the trigger and add it to a contactTriggerHandler class